content/installation/node/Node-cloud-foundry.md

```diff
@@ -0,0 +1,151 @@
+
+* Contact Contrast support for access to the service broker source code.
+
+* Deploy the service broker application:
+
+```bash
+cf push contrast-security-service-broker 
+```
+
+The service broker now appears in your Cloud Foundry console. The service broker doesn't offer any plans by default. You can configure plans using the `CONTRAST_SERVICE_PLANS` environment variable. If using Pivotal, you can also use the Pivotal Ops Manager to set the environment variables. If using Bluemix, you can click on the application, select **Runtime** and then **Environment Variables** to set the value. 
+
+* Refer to the following example to set the value through the command line:
+
+```
+    cf set-env contrast-security-service-broker CONTRAST_SERVICE_PLANS
+    " {
+            "ServicePlan1": {
+                 "name":"ServicePlan1",
+                 "teamserver_url":"https://yourteamserverurl.com",
+                 "username":"your_username",
+                 "org_uuid":"00000000-1111-2222-3333-000000000000",
+                 "api_key":"your_api_key",
+                 "service_key":"your_service_key"
+             },
+             "AnotherServicePlan":{
+                 "name":"AnotherServicePlan",
+                 "teamserver_url":"https://yourteamserverurl.com",
+                 "username":"your_username",
+                 "org_uuid":"00000000-1111-2222-3333-000000000001",
+                 "api_key":"your_api_key",
+                 "service_key":"some_other_service_key"
+                 }
+             } "
+```
+
+* After modifying the environment variable, restage your application.
+
+```
+cf restage contrast-security-service-broker
+```
+
+* The application also requires an environment variable for a username and a password:
+
+```bash
+cf set-env contrast-security-service-broker SECURITY_USER_NAME aSecureUsername
+cf set-env contrast-security-service-broker SECURITY_USER_PASSWORD aSecurePassword
+```
+
+* Create a service broker instance. (At least one service plan must be defined.) You must use the username and password configured above.
+
+```bash
+cf create-service-broker contrast-security-service-broker USER_NAME PASSWORD
+<URL of your application>
+```
+
+* All service brokers start off as private; you need to make it public.
+
+```bash
+cf enable-service-access contrast-security-service-broker
+```
+
+* Now that the service broker is working, create a service instance and bind it to the application. To create a service instance, run the following command:
+
+```
+cf create-service contrast-security-service-broker ServicePlan1 <name_of_service>
+```
+
+* To bind it to your application, run the following command:
+
+```
+cf bind-service <app_name> <name_of_service>
+```
+
+You should now see your application Contrast UI.
+
+## Contrast Service Broker Tile
+
+See [Contrast Service Broker Tile](https://docs.contrastsecurity.com/installation-javaserver.html#cloud) in the Java configuration documentation.
```

Comment:
@gmoore Do all of the Java instructions for the tile apply? If so, I'll just copy them over.

Reply:
Yes they do but I thought it better to just keep them in one place. Otherwise, when they do change, I'll probably forget to change both.